### PR TITLE
Fixed #34905 -- Corrected admin's main content element rendered in <main> tag.

### PR DESCRIPTION
**Pull request by ontowhee**
Switched the landmarks for div and main so the main does not wrap the sidebar.

### CODE REVIEW

**Review by knyghty — APPROVED** [review]
I tried it out locally and works here.

File: tests/admin_views/test_nav_sidebar.py [hunk 1]
```diff
@@ -42,7 +42,9 @@ def setUp(self):
 
     def test_sidebar_not_on_index(self):
         response = self.client.get(reverse("test_with_sidebar:index"))
-        self.assertContains(response, '<main class="main" id="main">')
+        self.assertContains(
+            response, '<main id="content-start" class="content" tabindex="-1">'
+        )
```
Comment (knyghty) [inline tests/admin_views/test_nav_sidebar.py:47]: I am actually not sure I remember the purpose of this line (and I think I might have written it) so I'm not sure if this should be testing the same (now div) element. That's probably more consistent with the existing test, anyway.

Comment (ontowhee) [reply tests/admin_views/test_nav_sidebar.py:47]: To me it seems like the test is ensuring that the sidebar is not rendered on the index (either the admin home page, or a section, as opposed to a model-group). It does make more sense to test the div, which would be consistent with the original intention. I'll update it.

**Comment by felixxm** [comment]
@ontowhee Thanks for this patch :+1: Please add test for this change, e.g.
```diff
diff --git a/tests/admin_views/tests.py b/tests/admin_views/tests.py
index 19877a933e..fe1086445e 100644
--- a/tests/admin_views/tests.py
+++ b/tests/admin_views/tests.py
@@ -1598,6 +1598,13 @@ class AdminViewBasicTest(AdminViewBasicTestCase):
         response = self.client.get(reverse("admin:login"))
         self.assertContains(response, '<header id="header">')
 
+    def test_main_content(self):
+        response = self.client.get(reverse("admin:index"))
+        self.assertContains(
+            response,
+            '<main id="content-start" class="content" tabindex="-1">',
+        )
+
 
 @override_settings(
     AUTH_PASSWORD_VALIDATORS=[

```

**Review by sabderemane — APPROVED** [review]
Nice work, thank you for your contribution @ontowhee ! 🎉 😄

**Comment by felixxm** [comment]
@ontowhee Thanks :+1: Welcome aboard :boat:

**Pull request merged**